src/status_im/integration_test.cljs

```diff
@@ -247,19 +193,17 @@
            (rf/dispatch-sync [:chat/navigate-to-chat chat-id])
            (is (= chat-id @(rf/subscribe [:chats/current-chat-id])))
            (logout!)
-           (rf-test/wait-for [::logout/logout-method]
-             (assert-logout))))))))
+           (rf-test/wait-for [::logout/logout-method])))))))
 
 (deftest delete-chat-test
   (log/info "========= delete-chat-test ==================")
   (rf-test/run-test-async
    (initialize-app!)
    (rf-test/wait-for
-     [:setup/initialize-view]
+     [:get-profiles-overview-success]
```

Comment:
can I ask why this event name has no context domain prepended to it, e.g -> `:profile/get-profiles-overview-success` ?

Reply:
idk probably because its not in profile context, but i could add it no problem

Reply:
i think i did this intentionally because it's not profile flow, yes its loads profile overviews but it's on app initialization step not in the profile flow

Reply:
ok but maybe we should have an `init` or `setup` as we had before prefacing it?

Reply:
yeah, i removed setup, because its not setup i guess, but init sounds fine